File: C Programming/Assignment/Assignment 7/sortarray.c
#include<stdio.h>
void main()
{
	int arr[5],i,j,temp;
	printf("Enter element of array");
	for(i=0;i<5;i++)
	{
		scanf("%d",&arr[i]);
	}
	
	for(i=0;i<4;i++)
	{
		for(j=0; j<4-i; j++)
		{
			if(arr[j]>arr[j+1])
			{
			
			temp=arr[j];
			arr[j]=arr[j+1];
			arr[j+1]=temp;
	    	}
		}
	}
	
	printf("\nSorted array");
	for(i=0;i<5;i++)
	{
		printf("%d ",arr[i]);
	}
}

File: Core Java/Assignment/16 Sep Array/SortArray.java
import java.util.Scanner;
import java.util.Arrays;
class SortArray{
  public static void main(String args[])
  {
    Scanner sc =new Scanner(System.in);
    int arr[]=new int[6];
    int i;
    System.out.println("Enter element of array");
    for(i=0;i<arr.length;i++)
    {
      arr[i]=sc.nextInt();
    }
    
   //sort array
   Arrays.sort(arr);
   
   System.out.println("Sorted Array");
  for(i=0;i<arr.length;i++)
  {
    System.out.print(arr[i]+" ");
  }
}
}
